corehq/apps/hqwebapp/tests/data/bootstrap5_diffs/javascript/hqwebapp/main.js.diff.txt

```diff
@@ -33,7 +33,7 @@
 -        $("input[type='submit']", $elem).button();
          $("input[type='text'], input[type='password'], textarea", $elem);
          $('.config', $elem).wrap('<div />').parent().addClass('container block ui-corner-all');
-
```

Comment:
@MartinRiese these files were not modified by this PR, curious how these got committed as diffs?

Reply:
@biyeun that is a good question. I just ran `./manage.py build_bootstrap5_diffs` after merging master in since I could not figure out how to resolve the conflict in the diff and it added those two. I don't have a good explanation for it. Do you want me to revert those two diff files?

Reply:
no worries, i did some rebasing and resetting of the commit history and made sure [this change](https://github.com/dimagi/commcare-hq/pull/33997/files#diff-015bfd39a2c6d7021ef5317b7bf398137d68ba8ca6c00e85d7654b7644236683R82) made it in at the proper moment. I think it was just crazy timing adding in this small change while I was also updating indentation of the file